tests/local_vespa/test_app/rules/synonyms_acronyms.sr

```diff
@@ -0,0 +1,15 @@
+@default
+
+ndc +> ="national determined contribution";
+
+[IEA] +> [IEA];
+[IEA] :- International Energy Agency;
```

Comment:
I tried the NDC example in both of these formats after seeing the former in a [Vespa PR](https://github.com/vespa-engine/vespa/pull/28463). Neither seemed to work.